docs/201_README.md

```diff
@@ -19,10 +19,10 @@ kubectl -n argocd get secret argocd-initial-admin-secret -o jsonpath="{.data.pas
 
 # Username: admin password: from output of command above
 WORKSHOP_USER="<username>"
-argocd --port-forward --port-forward-namespace argocd login
-argocd --port-forward --port-forward-namespace argocd repo add "https://github.com/$WORKSHOP_USER/ArgoCDRollouts"
-argocd --port-forward --port-forward-namespace argocd app create argo-rollouts --repo "https://github.com/$WORKSHOP_USER/ArgoCDRollouts" --path manifests/ArgoCD101-RolloutsController --dest-namespace argo-rollouts --dest-server https://kubernetes.default.svc
-argocd --port-forward --port-forward-namespace argocd app sync argo-rollouts
+argocd --port-forward-namespace argocd login
```

Comment:
actually for the login subcommand it is required to pass both flags.. probably a bug in the CLI tbh